### ISSUE TITLE
Left click on Icon should open the UI and right click should show context menu

### ISSUE THREAD

**Issue by Rexadev** [safing/portmaster-ui#338]
Now left or right it show menu

**Comment by dhaavi** [comment]
Which icon?

**Comment by Rexadev** [comment]
> Which icon?

Windows system tray on bottom right

**Comment by Kiefy** [comment]
I just wrote up a lengthy submission asking for exactly the same thing before I noticed somebody had already requested it.

One thing that's worth bringing up:
When you get a connection pop-up, it obscures the tray menu right click menu, so you either have to accept/dismiss all the notification pop-ups before you are able to open the main gui.  Allowing left-click to open the GUI would solve this issue, and who knows. it may still work when the right-click menu bugs out. as it **so very often** does.
